verify_mapping_quality.pl

```diff
@@ -126,19 +131,24 @@ sub verify_percent_coverage {
         # Run samtools depth and get results
         #------------------------------------#
         my $result = `samtools depth $file`;
+        
         #check for errors that occur while running samtools depth
 		die "Error: samtools depth exited with error while working with $file.\n" if (not defined $result);
 		
         # Now that we have the results...
         #--------------------------------#
         my $gap_length = get_gap_length($result, $min_depth);
-
+		
         my $line;
         if ( $gap_length == -1) {
             $line = "$name,0%";
         }
         else {
-            $line = sprintf "$name,%3.2f%%", (( $size - $gap_length ) / $size * 100);            
+        	#if the gap_length is negative, then it currently represents the negative total of all positions that
+        	#pass the threshold criteria.  Alter the value to represent the positions that do not pass criteria
+        	#in order to get proper output. 
+        	$gap_length = $size - (-$gap_length) if ($gap_length < 0);
```

Comment:
This is the conditional check that recognizes the gap_lengths generated when multiple reference contigs are used.